ci_config.json

```diff
@@ -192,90 +192,70 @@
   },
   "fscl-xalgorithms-c": {
     "fatal_warnings": true,
-    "build_options": [
-      "with_test=enabled"
-    ],
+    "skip_tests": true,
```

Comment:
You need to prefix your options with the project name (since it's used as a subproject):
```json
    "build_options": [
      "fscl-xalgorithms-c:with_test=enabled"
    ],
```

Reply:
how do I skip microsoft compiler? Havent gotten around to making it work with MSVC yet

Reply:
applied format for enabling my test cases